gatherers/censys.py

```diff
@@ -0,0 +1,86 @@
+import os
+import re
+import time
+import logging
+from censys import certificates
+
+# Register a (free) Censys.io account to get a UID and API key.
+uid = os.environ.get("CENSYS_UID", None)
+api_key = os.environ.get("CENSYS_API_KEY", None)
```

Comment:
I don't desire it from the censys-python library, personally -- but if you do, you should definitely open an issue.